CHANGELOG-3.6.md

```diff
@@ -0,0 +1,17 @@
+
+
+Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/etcd/blob/main/CHANGELOG-3.5.md).
+
+<hr>
+
+## v3.6.0 (TBD)
+
+See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
+
+### etcdctl v3
+
+- Add command to generate [shell completion](https://github.com/etcd-io/etcd/pull/13133).
+
+### etcdutl v3
```

Comment:
Before merging this, let's wait on https://github.com/etcd-io/etcd/pull/13142 to merge.

Reply:
#13142 Merged/